storage/interceptors/broker_create_catalog_interceptor.go

```diff
@@ -141,3 +148,13 @@ func brokerCatalogAroundTx(ctx context.Context, broker *types.ServiceBroker, fet
 
 	return nil
 }
+
+func addTenantToLabels(labels types.Labels, tenantKey string, tenantValue []string) types.Labels {
```

Comment:
this is not adding, it is setting. wouldn't setTenantLabel be a better name?

Reply:
or even setLabelValue, you pass in the key as well

Reply:
changed